src/payment/strategies/paypal-commerce/paypal-commerce-funding-key-resolver.ts

```diff
@@ -0,0 +1,34 @@
+export default class PaypalCommerceFundingKeyResolver {
+    resolve(methodId: string, gatewayId?: string) {
```

Comment:
Make the return value explicit? i.e.: `resolve(methodId: string, gatewayId?: string): string`

Reply:
Oh, sure. Thank you!